docs/user/ai_model_serving.md

```diff
@@ -0,0 +1,478 @@
+    "model_version": "1",
+    "outputs": [{
+            "name": "1463",
+            "shape": [1, 1000],
+            "datatype": "FP32",
+            "data": [ ....... ]
+        }]
+}
+```
+
+To verify the response, we'll use Python.
+We need to obtain the index of the highest element in the `.outputs[0].data`.
+
+```python
+import json
+
+with open('response.json') as f:
+    response = json.load(f)
+
+data = response["outputs"][0]["data"]
+argmax = data.index(max(data))
+print(argmax)
+```
+
+The output of the Python script we just ran should be `309`.
+We can validate it against [resnet's input data](https://github.com/openvinotoolkit/model_server/blob/main/client/common/resnet_input_images.txt):
+```
+../../../../demos/common/static/images/bee.jpeg 309
+```
+
+You can try querying the model using other images mentioned in the resnet's input data.
+
+#### Getting the model server's metrics
+
+To obtain Prometheus metrics of the model server simply make a request on `/metrics` endpoint:
+```sh
+curl "${DOMAIN}/metrics" \
+    --connect-to "${DOMAIN}::${IP}:"
+```
+
+Partial example output:
+```
+# HELP ovms_requests_success Number of successful requests to a model or a DAG.
+# TYPE ovms_requests_success counter
+ovms_requests_success{api="KServe",interface="REST",method="ModelReady",name="ovms-resnet50"} 4
+ovms_requests_success{api="KServe",interface="REST",method="ModelMetadata",name="ovms-resnet50",version="1"} 1
+```
+
+#### Other Inference Protocol endpoints
+
+To learn more about kserve endpoints see upstream documentation:
+- [V1 Inference Protocol](https://kserve.github.io/website/latest/modelserving/data_plane/v1_protocol/)
+- [Open Inference Protocol (V2)](https://kserve.github.io/website/latest/modelserving/data_plane/v2_protocol/)
+
+## Appendix
+### Overriding kserve configuration
+
+If you wish to override kserve settings, you need to make a copy of existing ConfigMap, tweak the desired settings, and overwrite the existing ConfigMap.
+
+Settings are stored in a ConfigMap named `inferenceservice-config` in the `redhat-ods-applications` namespace.
+Alternatively, you can copy the ConfigMap from `/usr/lib/microshift/manifests.d/001-microshift-ai-model-serving/kserve/inferenceservice-config-microshift-patch.yaml`.
+
+After tweaking it, you must apply the ConfigMap and restart kserve (e.g. by deleting Pod or scaling the Deployment down to 0 and back to 1).
+For RHEL For Edge and RHEL Image Mode systems, create a new manifest making sure it's applied after `/usr/lib/microshift/manifests.d/001-microshift-ai-model-serving`.
+
+### Limitations
+- AI Model Serving on MicroShift is only available on the x86_64 platform.
+- AI Model Serving on MicroShift supports a very specific subset of the RHOAI Operator components.
+- You must secure the exposed model server's endpoint (e.g. OAUTH2).
+- Not all model servers support IPv6.
+- Only OCI (ModelCar) model delivery system is tested and supported
+
+### Known issues
```

Comment:
@pmtk should we add here OVMS is not supported on IPv6? https://issues.redhat.com/browse/RHOAIENG-21756

Reply:
I agree, I missed that comment. Thanks.